cnvkit_call/cnvkit_call.xml

```diff
@@ -0,0 +1,190 @@
+        --normal-id "${baf_params.normal_id}"
+        #end if
+
+        #if $baf_params.min_var_depth:
+        --min-variant-depth ${baf_params.min_var_depth}
+        #end if
+
+        #if $baf_params.zygosity_freq:
+        --zygosity-freq ${baf_params.zygosity_freq}
+        #end if
+
+        "$input_file"
+    ]]></command>
+
+    <inputs>
+        <param name="input_file" type="data" format="tabular,tsv" 
+               multiple="false" label="File to process"/>
+        <conditional name="centering">
+            <param name="center_mode" type="select" label="Center Mode">
+                <option value="default">Default</option>
+                <option value="measure_of_center">Measure of Center</option>
+                <option value="manual">Manual Center</option>
+            </param>
+            <when value="measure_of_center">
+                <param name="measure" type="select" label="Measure">
+                    <option value="mean">Mean</option>
+                    <option value="median">Median</option>
+                    <option value="mode">Mode</option>
+                    <option value="biweight">Biweight</option>
+                </param>
+            </when>
+            <when value="manual">
+                <param name="offset" type="float" value="0.0"/>
+            </when>
+        </conditional>
+
+        <param name="filter" type="select" label="Merge flagged segements">
+            <option value="NA">No Merging</option>
+            <option value="ampdel">Merge using ampdel</option>
+            <option value="cn">Merge using cn</option>
+            <option value="ci">Merge using ci</option>
+            <option value="sem">Merge using sem</option>
+        </param>
+
+        <conditional name="method">
+            <param name="name" type="select" label="Calling Method">
+                <option value="NA">None</option>
+                <option value="threshold">Threshold</option>
+                <option value="clonal">Clonal</option>
+            </param>
+            <when value="threshold">
+                <param name="thresh1" type="float" value="-1.1" 
+                       label="Threshold 1"/>
+                <param name="thresh2" type="float" value="-0.25" 
+                       label="Threshold 2"/>
+                <param name="thresh3" type="float" value="0.2" 
+                       label="Threshold 3"/>
+                <param name="thresh4" type="float" value="0.7" 
+                       label="Threshold 4"/>
+            </when>
+        </conditional>
+
+        <param name="ploidy" type="integer" value="2" label="Ploidy"/>
+
+        <conditional name="purity">
+            <param name="known" type="select" label="Sample Purity Status">
+                <option value="unknown">Unknown</option>
+                <option value="known">Known</option>
+            </param>
+            <when value="known">
+                <param name="percentage" type="float" value="0.0" 
```

Comment:
want to confirm cnvkit expects a float for this param (instead of int)?

Reply:
The documentation gives the following example, so I believe so:
`cnvkit.py call Sample.cns -y -m clonal --purity 0.65 -o Sample.call.cns`